src/node_process_object.cc

```diff
@@ -91,6 +91,14 @@ MaybeLocal<Object> CreateProcessObject(Environment* env) {
     return MaybeLocal<Object>();
   }
 
+  // process._exitingAliasedUint32Array
+  Local<String> exiting_string =
+      FIXED_ONE_BYTE_STRING(env->isolate(), "_exitingAliasedUint32Array");
```

Comment:
Can we use a symbol property here instead?

Reply:
@joyeecheung yes, we could that but wouldn't it require us to expose the symbol as a value at a string index on the process object?

Reply:
ping @joyeecheung in case you missed the message

Reply:
> maybe util or process_object

Since `process_object` is the same as `process`, I think doing it on `util` or some other module is necessary, will do that

> better yet it can be a v8::Private

Does that have an advantage over using `v8::Symbol::New()`?

Reply:
@joyeecheung done, PTAL. Could you also clarify ^?

Reply:
@joyeecheung PTAL

Reply:
@joyeecheung